src/entities/purchase-params.ts

```diff
@@ -39,4 +40,15 @@ export interface PurchaseParams {
    * to the RC transaction as metadata.
    */
   metadata?: PurchaseMetadata;
+
+  /**
+   * Defines an optional override for the default branding appearance.
+   *
+   * This property can be used to customize the visual presentation or styling
+   * associated with the branding, replacing the default settings. When set, it
+   * supersedes the pre-configured appearance defined elsewhere in the system.
+   *
+   * @type {BrandingAppearance | undefined}
```

Comment:
I don't think you need to specify the type like this?

Reply:
I'll remove it!

Reply:
Done